docs/WEB_BUILD_INFO.md

```diff
@@ -0,0 +1,63 @@
+# How [https://flagd.dev](https://flagd.dev) is built
+
+The flagd project has a website at [https://flagd.dev](https://flagd.dev)
+
+This guide is intended so that potential contributors can understand how, where and when the website comes into being.
+
+## Website Structure
+
+The website is built on [mkdocs](https://www.mkdocs.org/) and the [material theme](https://squidfunk.github.io/mkdocs-material/).
+
+The [mkdocs.yml](../mkdocs.yml) file describes the high level metadata in addition to the main menu structure.
+
+The website is generated from files in the [web-docs](../web-docs/) folder.
+
+The website homepage is powered by the [index.md](../web-docs/index.md) file at the root of `/web-docs`
+
+## What happens during a Website-Related PR?
+
+Imagine you notice a typo on [https://flagd.dev](https://flagd.dev).
+
+You find the page inside [/web-docs](../web-docs/) and fix the typo and commit the change (remembering to sign your commit).
```

Comment:
```suggestion
Find the page inside [/docs](../docs/) and fix the typo then commit the change (remembering to sign your commit).
```

Clicking the pencil icon on the doc page may be the easy way to find the correct page.